examples/03-axpy-view/axpy-view-cxx.cc

```diff
@@ -48,13 +48,17 @@ extern "C" {
     view_type y = **v_y;
     view_type x = **v_x;
 
+    double d_alpha = *alpha;
     Kokkos::parallel_for( "axpy", y.extent(0), KOKKOS_LAMBDA( const size_t idx)
     {
-      y(idx) += *alpha * x(idx);
+        y(idx) += d_alpha * x(idx);
```

Comment:
Why did you change the indentation?

Reply:
ok, sorry, I restored the original indentation.